Behavioral.Automation.Bindings.UI/Abstractions/IInputElement.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Behavioral.Automation.Bindings.UI.Abstractions;
+
+public interface IInputElement : IWebElement
+{
+    public Task TypeAsync(string text);
```

Comment:
I think `InputTextAsync` is clearer here

Reply:
Changed, thanks!